clients/src/test/java/org/apache/kafka/clients/consumer/internals/CommitRequestManagerTest.java

```diff
@@ -123,6 +126,47 @@ public void setup() {
         this.props.put(VALUE_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class);
     }
 
+    @Test
+    public void testOffsetFetchRequestStateToStringBase() {
+        final long retryBackoffMs = 10;
+        final long retryBackoffMaxMs = 100;
+        final long expirationTimeMs = 1000;
+        ConsumerConfig config = mock(ConsumerConfig.class);
+        CommitRequestManager.MemberInfo memberInfo = new CommitRequestManager.MemberInfo();
+
+        this.commitRequestManager = new CommitRequestManager(
+                time,
+                logContext,
+                subscriptionState,
+                config,
+                coordinatorRequestManager,
+                offsetCommitCallbackInvoker,
+                "groupId",
+                Optional.of("groupInstanceId"),
+                metrics);
+
+        this.offsetFetchRequestState = commitRequestManager.new OffsetFetchRequestState(
+                mock(Set.class),
+                retryBackoffMs, retryBackoffMaxMs, expirationTimeMs,
+                memberInfo
+        );
+
+        this.requestState = new RequestState(
```

Comment:
this should be `new RetriableRequestState` since offsetFetchRequestState implements the retriable one?